src/test/java/ikysil/training/ws/api/v1/OrdersResourceTest.java

```diff
@@ -24,27 +30,170 @@
 @RunWith(SpringRunner.class)
 public class OrdersResourceTest {
 
+    @Autowired
+    OrderDao dao;
+
     public static final String BASE_URL = "/ws/api/v1/orders/";
 
     @LocalServerPort
     private int port;
 
+    private OrderDto order1 = new OrderDto("ikl11444", "twenty one", "none", "3129", "firstuser@gmail.com");
+    private OrderDto order2 = new OrderDto("qwe32577", "twenty two", "none", "4024", "seconduser@gmail.com");
+    private OrderDto newOrder = new OrderDto("qwe32577","thirty three", "noneone", "2440", "newuser@gmail.com");
+
     @Before
     public void setUp() {
         RestAssured.basePath = BASE_URL;
         RestAssured.port = port;
+        dao.saveOrder(order1);
+        dao.saveOrder(order2);
     }
 
     @Test
-    // TODO rename test
-    public void test() {
-        OrderDto request = new OrderDto();
-
+    public void savingOKTest() {
+        given().contentType(ContentType.JSON)
+                .body(order1, ObjectMapperType.JACKSON_2)
+                .when()
+                .post()
+                .then()
+                .statusCode(200);
         given().contentType(ContentType.JSON)
-                .body(request, ObjectMapperType.JACKSON_2)
+                .body(order2, ObjectMapperType.JACKSON_2)
                 .when()
                 .post()
                 .then()
-                .statusCode(INTERNAL_SERVER_ERROR.getStatusCode());
+                .statusCode(200);
+    }
+
+    @Test
+    public void saveOrderTest(){
+        OrderDto tempOrder = new OrderDto("666","thirty six", "nothing", "2660", "newuser66@gmail.com");
+        assertEquals(2, dao.findAll().size());
```

Comment:
better/more readable/flexible 
```
assertThat(dao.findAll()).hasSize(2)[.contains*()....];
```
see similar assertions and change them as well

Reply:
![screen shot 2018-09-14 at 09 31 17](https://user-images.githubusercontent.com/31449825/45533605-f5e90500-b800-11e8-8668-b693ea279ffa.png)
That's why